src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseInviteService.cs

```diff
@@ -78,6 +78,23 @@ public class ReleaseInviteService : IReleaseInviteService
                 });
         }
 
+        public async Task<Either<ActionResult, Unit>> RemoveByPublication(
```

Comment:
Doesn't seem like we've added any tests for this?

Reply:
Missed this - my bad. Added some tests